mods/pageviews.js

```diff
@@ -146,6 +126,23 @@ Object.keys(viewCountColumnsForArticleFlat).forEach(function(k) {
     tableSchemas.articleFlat.attributes[viewCountColumnsForArticleFlat[k]] = 'int';
 });
 
+var notFoundCatcher = function(e) {
+    if (e.status === 404) {
+        throw new rbUtil.HTTPError({
```

Comment:
Should be enough to modify `e` directly, i.e. modify its `e.body.description` field.